### ISSUE TITLE
Version 5.2.0, creating make.inc, running make fails, superlu_dist_config.h not created

### ISSUE THREAD

**Issue by zerothi** [xiaoyeli/superlu_dist#12]
I cannot get SuperLU-dist 5.2.0 to to compile.

I am using simple makefiles (no Cmake) and I have created a make.inc, just as I would for 5.1.3. However, when I run "make" I find this error
```
( cd INSTALL; make )
make[1]: Entering directory '.../superlu_dist-5.2.0/INSTALL'
mpicc -c ../SRC/superlu_timer.c
In file included from ../SRC/superlu_timer.c:24:0:
../SRC/superlu_defs.h:69:10: fatal error: superlu_dist_config.h: No such file or directory
 #include "superlu_dist_config.h"
          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:20: superlu_timer.o] Error 1
make[1]: Leaving directory '.../superlu_dist-5.2.0/INSTALL'
make: *** [Makefile:27: install] Error 2
```
However, `superlu_dist_config.h.in` exists but isn't anywhere to be found in the Makefiles?

The file seem new in 5.2.0?

**Comment by xiaoyeli** [comment]
Just got this fixed in the latest v5.2.1.  Please check out.

-Sherry Li

**Comment by zerothi** [comment]
I have to re-open the issue.

The target is not created in the correct order, the error message is the same because `superlu_dist_config.h` needs to be created in SRC, but make will first run in INSTALL, so the target is never created (one can of course manually go into SRC, do `make config_h`, go back and re-run make, but...
